src/common/settings.js

```diff
@@ -2,6 +2,7 @@
 
 const fs = require('fs');
 const settingsVersion = 1;
+const app = require('electron').app;
```

Comment:
The js is tested in `test/specs/settings_test.js` by mocha. Please consider not using `require('electron')` here. For example, add an arg for `upgrade()`.